plugins/statistics.koplugin/main.lua

```diff
@@ -1568,6 +1568,10 @@ function ReaderStatistics:deleteBook(id_book)
     conn:close()
 end
 
+function ReaderStatistics:onPageUpdate(pos, pageno)
```

Comment:
@poire-z Should it be:
`ReaderStatistics:onPosUpdate(pos, pageno)`

Reply:
Has to be, otherwise it makes no sense.

Reply:
Indeed :| Fortunatly, it is overwritten by the next one (good I put that one before the good one :) PR'ing a correction. Thanks!

Reply:
I'm actually slightly surprised luacheck didn't complain about this.